src/core.js

```diff
@@ -510,7 +511,7 @@
           global.clearInterval(connect.core.iframeRefreshInterval);
           connect.core.iframeRefreshInterval = null;
           connect.core.getPopupManager().clear(connect.MasterTopics.LOGIN_POPUP);
-          if (params.loginPopupAutoClose && connect.core.loginWindow) {
+          if (params.loginPopup && params.loginPopup.autoClose && connect.core.loginWindow) {
```

Comment:
doesn't this change the existing behavior since `autoClose` defaults to false.

Reply:
This logic is exactly the same as it was before, there is a just an extra check to make sure `loginPopup` exists to avoid a reference error since `autoClose` is not a property of it.

Reply:
This is going to change, depending on your answer to [this question](https://github.com/amazon-connect/amazon-connect-streams/pull/192#discussion_r408540330).

Reply:
since `loginPopupAutoClose` has been out for awhile, I'd recommend to make this backward compatible, by checking if `loginPopupAutoClose` was set to `true` as well